backend/modules/evaluation/domain/service/expt_run_item_impl.go

```
@@ -215,6 +215,13 @@ func (e *ExptItemEvalCtxExecutor) buildExptTurnEvalCtx(ctx context.Context, turn
 	for k, v := range eiec.Event.Ext {
 		etec.Ext[k] = v
 	}
+	// 从 ExptItemResult 中获取 Ext 字段并合并到 etec.Ext
+	itemResults, err := e.ItemResultRepo.BatchGet(ctx, spaceID, eiec.Event.ExptID, []int64{eiec.Event.EvalSetItemID})
+	if err == nil && len(itemResults) > 0 && itemResults[0].Ext != nil {
+		for k, v := range itemResults[0].Ext {
+			etec.Ext[k] = v
```

[P1] Silent error swallow on BatchGet; add logging/fallback. Currently `err` is ignored, which hides DB issues and weakens observability. Suggest logging the error and proceed with event-only Ext.

```go
if err != nil {
    logs.CtxWarn(ctx, "BatchGet item result failed: %v", err)
}
```